.github/workflows/operator_upgrade.yaml

```diff
@@ -43,10 +46,13 @@ jobs:
         uses: ./server/.github/actions/setup-java
 
       - name: Build Infinispan
+        id: infinispan
         working-directory: server
         run:  |
           ./mvnw install -DskipTests -am -pl server/runtime
           SERVER_VERSION=$(./mvnw -q -Dexec.executable=echo -Dexec.args='${project.version}' --non-recursive exec:exec)
+          OPERAND_VERSION=$(echo ${SERVER_VERSION} | cut -d '-' -f 1)
+          echo "version=${OPERAND_VERSION}" >> ${GITHUB_OUTPUT}
```

Comment:
I don't know if it is the case also to log the `OPERAND_VERSION` variable.

Reply:
I didn't explicitly log this as we can already determine the Operand that is added by the "Log CSV Deployment" step in the upgrade job. In the printed Deployment, there will only be one image with an image starting with "localhost:5001/server" and that is associated with the passed `$OPERAND_VERSION`.

Reply:
makes sense